tests/runner/kube_testplatform.go

```diff
@@ -43,11 +45,33 @@ func (c *KubeTestPlatform) tearDown() error {
 		log.Errorf("Failed to tearDown AppResources. got: %q", err)
 	}
 
+	if err := c.ComponentResources.tearDown(); err != nil {
+		log.Errorf("Failed to tearDown ComponentResources. got: %q", err)
```

Comment:
nit: as per Dapr tracing conventional, start tracing with "lowercase"...like "failed to teardown..."

Reply:
I corrected the messages